.nvmrc

```diff
@@ -1 +1 @@
-lts/erbium
+v16
```

Comment:
Oops, this was a mistake to include in this PR. (But if you are relying on `nvm` for npm as well as Node, it's helpful as this repo is already on package-lock.json format 2 from npm v7 whereas Node lts/erbium pulls in npm v6.)